src/index.tsx

```diff
@@ -186,18 +190,23 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
     const { withHandle } = this.props;
 
     if (!withHandle) {
-      return 20;
+      return 0;
```

Comment:
By removing the 20px, the modal is now stuck to the top of the screen without margin. It was intentional to have this space when the handle is not shown. Even though, I'm not again a props to change this margin